sci-electronics/systemc/systemc-2.3.0-r1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=4
 
-inherit eutils multilib toolchain-funcs autotools-utils
+inherit multilib toolchain-funcs autotools-utils
```

Comment:
it is not unused. The ebuild uses `epatch` which used to reside in `eutils`. The correct solution would be to replace it with `epatch` eclass.